src/project.ts

```diff
@@ -3,6 +3,25 @@ import glob from 'glob';
 import { Package, readPackage } from './package';
 import { PackageManifestFieldNames } from './package-manifest';
 import { getRepositoryHttpsUrl } from './repo';
+import { SemVer } from './semver';
+
+/**
+ * The release version of the root package of a monorepo extracted from its
+ * version string.
+ *
+ * @property ordinaryNumber - The number assigned to the release if it
```

Comment:
I tossed around a few names in my head and went with "ordinary" as an opposite of "backport". Is this a good name?

Reply:
Seems like a good name to me!